styles.css

```diff
@@ -458,3 +469,414 @@ form {
+    display: none;
+  }
+
+  .container > .flip-img {
+    grid-column: 2 / 3;
+  }
+
+  .client-name {
+    height: 50%;
+    align-self: end;
+    margin: 2%;
+    font-size: 2.78vw;
+  }
+
+  .container > ul {
+    margin: 7%;
+    margin-right: 0;
+    height: 3vh;
+    margin-top: 0;
+    margin-bottom: 0;
+    align-self: center;
+  }
+
+  .container > ul > .name {
+    font-size: 1.25vw;
+  }
+
+  .about-project {
+    margin: 2%;
+  }
+
+  .buttons {
+    margin: 2%;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  .see-project {
+    margin-left: 7%;
+    padding: 0 1% 0 1%;
+    width: 10vw;
+    align-self: start;
+    font-size: 1.39vw;
+  }
+
+  .contact {
+    width: 50%;
+  }
+
+  .logo,
+  .about-project {
+    font-size: 1.112vw;
+    line-height: 1.7;
+    text-decoration: none;
+  }
+
+  .greeting {
+    font-size: 3.334vw;
+    grid-row: 2/3;
+  }
+
+  .container > ul > li {
+    font-size: 1.2vw;
+    font-weight: 50;
+    margin: 0;
+  }
+
+  .second-text {
+    grid-row: 4/6;
+  }
+
+  .buttons > ul > li {
+    margin: 2%;
+    font-size: 0.85vw;
+    font-weight: 50;
+    padding: 0.07rem 0.3rem 0.07rem 0.3rem;
+    border-radius: 0.3rem;
+  }
+
+  .buttons > ul {
+    margin: 0;
+  }
+
+  .dropdown-content > a > img {
+    padding: 0;
+    width: 4.5vw;
+    grid-row: 1/2;
+    grid-column: 1/2;
+  }
+
+  .socials > ul > li > a > img {
+    width: 1.4vw;
+  }
+
+  /* about myself */
+  #headline-two {
+    display: grid;
+    grid-template-columns: repeat(2, 40% 60%);
+    grid-template-rows: repeat(9, 5vh);
+    height: 55vh;
+    padding: 8.2% 8.2% 4.2% 8.2%;
+    background-color: #fff;
+    border-radius: 0 20% 0 0;
+    width: 86vw;
+  }
+
+  .wrapper-contact > div > h2 {
+    font-size: 3vw;
+  }
+
+  #headline-two > h2 {
+    grid-row: 8/9;
+  }
+
+  #headline-two > .socials {
+    grid-row: 9/10;
+    align-self: center;
+  }
+
+  #about-myself {
+    height: 60vh;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-image: none;
+    background-color: #f7f7f9;
+  }
+
+  #resume {
+    grid-row: 10/11;
+    width: 13vw;
+    padding: 0;
+  }
+
+  .dropbtn {
+    font-size: 1.4vw;
+    padding: 1rem;
+  }
+
+  .dropdowns {
+    grid-column: 2 / 3;
+    grid-row: 2/6;
+  }
+
+  .list-of-dropdowns {
+    margin: 0;
+  }
+
+  .dropdown-content {
+    display: none;
+  }
+
+  .list-of-dropdowns > li:hover .dropdown-content {
+    display: flex;
+    flex-direction: row;
+    justify-content: start;
+  }
+
+  .dropdown-content > a {
+    font-size: 1.3vw;
+    height: 7.5vw;
+    width: 7.5vw;
+    display: grid;
+    grid-template-rows: 70% 30%;
+    padding: 2%;
+  }
+
+  .dropdown-content > a > span {
+    grid-row: 2/3;
+    grid-column: 1/3;
+    font-weight: 250;
+    color: #253858;
+  }
+
+  /* contact form */
+
+  #contact-form {
+    height: 80vh;
+    margin: 0;
+    padding: 0;
+    background-color: #fff;
+    margin-top: 5%;
+  }
```

Comment:
- Please add some padding to the p tag here and the "get in touch" button is also different from the Figma design. Kindly makes these changes just as the previous reviewer has also mentioned.

Figma design
![image](https://github.com/agneta1991/Portfolio/assets/89091822/9e435ab5-0a17-4fe7-813c-94bd0684c240)

Your design
![image](https://github.com/agneta1991/Portfolio/assets/89091822/710a2025-9e23-473f-9fbc-98b728639aa3)